### PR TITLE
sickle: import from homebrew/science

### PR DESCRIPTION
**Pull request by sjackman**
Please update https://github.com/Homebrew/homebrew-science/blob/master/tap_migrations.json if this PR is merged.

### CODE REVIEW

File: Formula/sickle.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Sickle < Formula
+  desc "Windowed adaptive trimming for FASTQ files using quality"
+  homepage "https://github.com/najoshi/sickle"
+  url "https://github.com/najoshi/sickle/archive/v1.33.tar.gz"
+  sha256 "eab271d25dc799e2ce67c25626128f8f8ed65e3cd68e799479bba20964624734"
+
+  def install
+    system "make"
+    bin.install "sickle"
```
Comment (ilovezfs) [inline Formula/sickle.rb]: It would be good to request a make install target if we haven't already

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks again @sjackman!

File: Formula/sickle.rb [hunk 2]
```diff
@@ -17,7 +16,7 @@ def install
       +
       IIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIIII0000000000
     EOS
-    output = shell_output("#{bin}/sickle se -f test.fastq -t sanger -o /dev/stdout")
-    assert_match "GTGTC\n", output
+    cmd = "#{bin}/sickle se -f test.fastq -t sanger -o /dev/stdout"
+    assert_match "GTGTC", shell_output(cmd).chomp
```
Comment (sjackman) [inline Formula/sickle.rb]: This change no longer tests what I wanted to test, that after running Sickle, the line now ends in `GTGTC`. If Sickle hadn't modified the input at all, this test would still pass. Please revert this change.

**Comment by sjackman** [comment]
Thanks for merging, Joe. Please see my comment regarding the test.

File: Formula/sickle.rb (hunk 2 above)
Comment (sjackman) [reply Formula/sickle.rb]: You could use…
```ruby
assert_equal output.lines[1].chomp, "AGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC"
```
